### DISCUSSION TITLE
How can I add new parameter in values.yaml

### DISCUSSION THREAD

**Discussion by swathida-byte** [actions/actions-runner-controller#2027] [Questions]
I need to add args parameter under https://github.com/actions-runner-controller/actions-runner-controller/blob/master/charts/actions-runner-controller/values.yaml#L51

**Comment by samus-aran** [comment]
We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
